HZ: 20.0
TARGET_VELOCITY: 0.8
MAX_VELOCITY: 1.0
MIN_VELOCITY: 0.0
MAX_OMEGA: 0.8
MAX_ACCELERATION: 1.0
MAX_ANG_ACCELERATION: 2.0
VELOCITY_RES: 0.05
OMEGA_RES: 0.05
SIM_TIME: 5.0
TO_GOAL_COST_GAIN: 0.5
SPEED_COST_GAIN: 1.0
OBSTACLE_COST_GAIN: 2.0
#ANGLE_RESOLUTION: 0.0087
#MAX_DIST: 10.0
#GOAL_THRESHOLD: 0.3
#TURN_DIRECTION_THRESHOLD: 0.1